src/commands/tail/mod.rs

```diff
@@ -184,8 +211,8 @@ pub fn command(args: &[&str], binary_path: &PathBuf) -> Command {
     };
 
     c.args(args);
-    // Let user read cloudflared process logs iff RUST_LOG=debug.
-    if !log_enabled!(Debug) {
+    // Let user read cloudflared process logs iff RUST_LOG=info.
```

Comment:
this should probably be enabled with `--verbose`

Reply:
https://github.com/cloudflare/wrangler/issues/1175